From b1a14b3056873e639b0a61b379a23cc18438acd5 Mon Sep 17 00:00:00 2001
From: LuoXiaoTan <lxt@rock-chips.com>
Date: Thu, 13 Apr 2017 20:27:30 -0700
Subject: [PATCH] arm64: dts: rk3399-sapphire: add HDMIIn sound dev

sable rt5651 simple-audio-card and we will write
a machine driver for rt5651 tc358749x HDMIIn function.
And add rt5651-tc358749x-sound for HDMIIn function

Change-Id: I983fa9171bfa1fb3bf0ab492354e471046c08e93
Signed-off-by: LuoXiaoTan <lxt@rock-chips.com>
---
 .../rockchip/rk3399-sapphire-excavator-edp-android-6.0.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-android-6.0.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-android-6.0.dts
index 03afa71e3a45..53447de2ecc0 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-android-6.0.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-android-6.0.dts
@@ -54,6 +54,13 @@
 	};
 
 	rt5651-sound {
+		status = "disabled";
+	};
+
+	hdmiin-sound {
+		compatible = "rockchip,rockchip-rt5651-tc358749x-sound";
+		rockchip,cpu = <&i2s0>;
+		rockchip,codec = <&rt5651 &rt5651 &tc358749x>;
 		status = "okay";
 	};
 };
-- 
2.35.3

